app/imports/components/dashboards/MainDashboard.jsx

```diff
@@ -1,13 +1,22 @@
 import React, { Component } from 'react';
 
+import { ListWorkersContainer } from '/imports/containers/ListWorkersContainer.jsx';
+import { AddWorker } from '/imports/components/workers/AddWorker.jsx';
+
 export class MainDashboard extends Component {
   render() {
     return(
-      <div className="grid-row">
-        <section className="grid-item item-s-12">
-          <h1>Dashboard</h1>
-        </section>
-      </div>
+      <section id="dashboard">
+        <div className="grid-row">
+          <div className="grid-item item-s-12">
+            <h1>Dashboard</h1>
+          </div>
+        </div>
+
+        <ListWorkersContainer />
```

Comment:
Oh but either this needs to be conditionally shown according to user role or put in its own route which is admin user only. I would vote for own route